# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======

MergePointsActionGoal action_goal
MergePointsActionResult action_result
MergePointsActionFeedback action_feedback
